spectree/plugins/starlette_plugin.py

```diff
@@ -71,13 +71,18 @@ def register_route(self, app):
         await response(scope, receive, send)
 
     def find_routes(self):
+        from starlette.routing import Mount
         routes = []
 
         def parse_route(app, prefix=''):
             for route in app.routes:
                 if route.path.startswith(f'/{self.config.PATH}'):
                     continue
 
+                if isinstance(route, Mount):
+                    parse_route(route, prefix=f'{prefix}{route.path}')
+                    continue
+
                 func = route.endpoint
```

Comment:
```suggestion
                func = route.app
```
Sorry, my bad. This should fix this problem. No need to check if it's an instance of `Mount`

Reply:
Cool, I've pushed this change instead.